camera_frame: camera_depth_optical_frame
deployed_template_cloud_topic: /deployed_template_cloud
filtered_source_cloud_topic: /filtered_source_cloud
filtered_template_cloud_topic: /filtered_template_cloud
initial_pitch: 0.0
initial_roll: 0.0
initial_x: 0.0
initial_y: 0.0
initial_yaw: 0.0
initial_z: 0.0
pcd_file: /home/std/workspace/test_pkg_ws/src/docking_pose_estimator/template/deploy/box_nerf_origin.pcd
pointcloud_topic: /camera/depth/points
qos_overrides./parameter_events.publisher.depth: 1000
qos_overrides./parameter_events.publisher.durability: volatile
qos_overrides./parameter_events.publisher.history: keep_last
qos_overrides./parameter_events.publisher.reliability: reliable
robot_frame: base_footprint
save_command_topic: /save_pointcloud_command
save_path: /home/std/workspace/test_pkg_ws/src/docking_pose_estimator/template/deploy/
template_x_max: 3.0
template_x_min: 0.0
template_y_max: 1.0
template_y_min: -1.0
template_z_max: 2.0
template_z_min: 0.0
use_ndt: false
use_sim_time: false
voxel_size: 0.05
x_max: 1.0
x_min: 0.0
y_max: 1.0
y_min: -1.0
z_max: 2.0
z_min: 0.6
